### ISSUE TITLE
Upgrade to using Python 3.9

### ISSUE THREAD

**Issue by thezoggy** [impredicative/irc-rss-feed-bot#80]
upgraded python to latest, so on 3.9.2 now and found that irc-rss-feed-bot installer breaks due to hext not being able to be installed during ` pip install -r requirements.txt` as there is no 3.9 support at this time for it.

```
irc-rss-feed-bot$ pip install hext
ERROR: Could not find a version that satisfies the requirement hext
ERROR: No matching distribution found for hext
```

hext is aware - https://github.com/html-extract/hext/issues/18

just noting it here for others, feel free to close

**Comment by impredicative** [comment]
Noted. I encourage using the published Docker image which is a known working image. The included Dockerfile should work too.

**Comment by thezoggy** [comment]
> Hext is now available for Python 3.9

**Comment by impredicative** [comment]
The bot is currently being tested with Python 3.9 in the `master` branch. Some other prior changes are also being tested, and this will take time. A new release will be created whenever the testing has concluded. Thus far it's working fine with Python 3.9.

**Comment by impredicative** [comment]
Released in 0.11.0.
